woltka/tests/test_file.py

```diff
@@ -119,6 +119,13 @@ def test_id2file_from_dir(self):
         obs = id2file_from_dir(self.tmpdir)
         exp = {x: '{}.faa'.format(x) for x in ids}
         self.assertDictEqual(obs, exp)
+
+        # skip subdirectory
+        sdir = join(self.tmpdir, 'im_dir')
+        makedirs(sdir)
+        obs = id2file_from_dir(self.tmpdir)
+        self.assertDictEqual(obs, exp)
```

Comment:
Would suggest adding a comment that explains that this is testing that subdirectories are skipped due to not being files (purpose of the test wasn't clear to me immediately).

Also, you may want to consider how this would work when there are files within the subdirectory (e.g. `im_dir/abc.faa`). Should those be included in the `id2file` mapping, or not? In either case, you could add to the test here to check how this case is handled.

Reply:
Only the top-level directory is searched. Therefore anything in subdirectories is ignored. I added this comment to the docstring of `id2file_from_dir` (see above). I also added a comment to this test function.